build.gradle.kts

```diff
@@ -130,9 +152,18 @@ tasks {
         )
     }
 
+    register<Download>("downloadBufBeta") {
+        description = "Download beta version of the Buf CLI."
+        src("https://github.com/bufbuild/buf/releases/download/v1.32.0-beta.1/buf-${bufOS()}-${bufArch()}${bufExt()}")
```

Comment:
Once we release a new version of the CLI, I'll remove all this and update the version used by the tests to be the new version. We didn't publish a Maven version of the beta so this is just a temporary workaround to download the pre-built CLI from GitHub releases.